cartofacade/capi_test.go

```diff
@@ -362,14 +362,15 @@ func TestCGoAPIWithIMU(t *testing.T) {
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, pvcl, test.ShouldNotBeNil)
 
-		cfg := GetTestConfig("mylidar", "myIMU", "", true)
+		cfg := GetTestConfig("mylidar", "", "", true)
 
 		// test invalid IMU enabling configuration
-		algoCfg := GetTestAlgoConfig(false)
+		algoCfg := GetTestAlgoConfig(true)
```

Comment:
I don't think we'll have to test this for the odometer, since we most likely won't have a `use_odometer_data` field in `viam_carto_algo_config`, in which case we won't be able to check this case. If we end up having this field, though, then we'll still have to have a separate test for the odometer. We should not change this to `movement_sensor`, since at this level we already separated the movement sensor into IMU and odometer. 

Happy to discuss in person if this doesn't make much sense.